examples/quantum_die.py

```diff
@@ -28,47 +28,43 @@ def qubits_needed(n):
     """
     return int(math.ceil(math.log(n, 2)))
 
+def qvm(n):
+    """
+    Connect to a QVM that simulates n sides.
+    """
+    if qubits_needed(n) == 9:
+        return qvm = get_qc('9q-square-qvm')
+    else:
+        return qvm = get_qc('%(qubits)dq-qvm' % {"qubits": qubits_needed(n)})
+
 def die_program(n):
     """
     Generate a quantum program to roll a die of n faces.
     """
-    prog = pq.Program()
+    prog = Program()
     qubits = qubits_needed(n)
```

Comment:
can we rename `qubits` to `n_qubits`. Idiomatically, qubits should be a list of qubit ids